data/os/RedHat/8.4.yaml

```
@@ -0,0 +1,12 @@
+---
+
```

Would there be any logic in setting these outdated RHEL `8.2` & `8.4` versions to default to enabling `qualys_eus_reporting`?

```suggestion

profile_audit::qualys_eus_reporting::enabled: true

```

**Probably not**, because `CentOS`, `Rocky`, etc. could also be referencing these `os/RedHat/8.x.yaml` files.

---

I was leaning towards no. First for the reason you mention about Centos/Rocky, but also since the 8.2, 8.4, and other 8.x yaml files will always be set in the module. They just act as placeholders to set the strings for whatever EUS repos we might need.